lib/core/include/rodsErrorTable.h

```diff
@@ -233,6 +233,7 @@ NEW_ERROR(SYS_SOCK_WRITE_ERR,                          -161000)
 NEW_ERROR(SYS_SOCK_CONNECT_ERR,                        -162000)
 NEW_ERROR(SYS_OPERATION_IN_PROGRESS,                   -163000)
 NEW_ERROR(SYS_REPLICA_DOES_NOT_EXIST,                  -164000)
+NEW_ERROR(SYS_UNKNOWN_ERROR,                           -165000)
```

Comment:
`SYS_INTERNAL_ERR` may communicate the same amount of information if we want to avoid adding an new error.

Reply:
This new error code was introduced during a group code review for cases where devs choose to use -1 to indicate an error occurred, but aren't sure what happened.